draft-ietf-quic-transport.md

```diff
@@ -697,17 +698,15 @@ as load balancers, to locate and use it.
 
 The client MUST choose a random connection ID and use it in Client Initial
 packets ({{packet-client-initial}}) and 0-RTT packets ({{packet-protected}}).
-If the client has received any packet from the server, it uses the connection ID
-it received from the server for all packets other than 0-RTT packets.
 
 When the server receives a Client Initial packet and decides to proceed with the
 handshake, it chooses a new value for the connection ID and sends that in a
 Server Cleartext packet.  The server MAY choose to use the value that the client
 initially selects.
 
 Once the client receives the connection ID that the server has chosen, it uses
-this for all subsequent packets that it sends, except for any 0-RTT packets,
-which all have the same connection ID.
+this for all subsequent 1-RTT packets ({{packet-protected}}) and only 1-RTT
+packets that it sends.
```

Comment:
Actually, this conflicts with # 558, which states that "Client Cleartext" also has SCID.  I'll fix that, too, and repost.